synapse/tchap/event_rules.py

```diff
@@ -121,6 +124,42 @@ def check_event_allowed(self, event, state_events):
 
         return ret
 
+    def _on_rules_change(self, event, state_events):
+        new_rule = event.content.get("rule")
+
+        # Check for invalid values.
+        if (
+            new_rule != ACCESS_RULE_DIRECT
+            and new_rule != ACCESS_RULE_RESTRICTED
+            and new_rule != ACCESS_RULE_UNRESTRICTED
```

Comment:
You can do `new_rule not in (ACCESS_RULE_DIRECT, ACCESS_RULE_RESTRICTED, ACCESS_RULE_UNRESTRICTED)` fwiw

Reply:
Ah, good point, thanks